support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -432,6 +436,14 @@ footer[role="contentinfo"] a {
     }
   }
 
+  .gu-content__blurb--help-variant {
+    max-width: 404px;
+    p {
```

Comment:
when I was doing the css strpping back, I noticed a few where we were having nested element names e.g. h1, p, rather than class names, so I removed.  Is it normal to have them in or not?